src/__tests__/Layout-test.js

```diff
@@ -1586,5 +1587,78 @@ describe('Layout', function() {
     );
   });
 
+});
+
+describe('Layout alignContent', function() {
```

Comment:
Most likely, going to check that, I didn't know they were supposed to be transpiled to C ;)